### PR TITLE
File tree restructure

### PR DESCRIPTION
**Pull request by bgribz**
The result of our Friday afternoon Mario Kart session.

- Massively cleans up the file tree
- Demonstrates how to use the container pattern
- Demonstrates cleaner imports/exports of components

There are still some improvements to be made, but I think this a good starting point for everyone to work off and move forward with.

Thoughts?

### CODE REVIEW

File: web-app/src/components/RequestHoliday/container.js [hunk 1]
```diff
@@ -86,7 +83,7 @@ class RequestHoliday extends React.Component {
             }
 
             requestHolidays(holidays);
-
+import Swal from 'sweetalert2';
```
Comment (William-Unosquare) [inline web-app/src/components/RequestHoliday/container.js]: This needs removed and put at the top of the file

File: web-app/src/components/TakenLeave/container.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        })
+    }
+
+    isDateInThePast(date){
+        return Moment(date).isBefore(new Date());
+    }
+
+    render() {
+      return (
+        <Wrapped takenHolidays={this.state.takenHolidays} {...props} />
```
Comment (William-Unosquare) [inline web-app/src/components/TakenLeave/container.js]: Change `{...props}` to `{...this.props}`

File: web-app/src/components/BookedLeave/container.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        })
+    }
+
+    isDateInTheFuture(date){
+        return Moment(date).isAfter(new Date());
+    }
+
+    render() {
+      return (
+        <Wrapped bookedHolidays={this.state.bookedHolidays} {...props} />
```
Comment (William-Unosquare) [inline web-app/src/components/BookedLeave/container.js]: Change `{...props}` to `{...this.props}`

**Review by William-Unosquare — CHANGES_REQUESTED** [review]
Some bug fixes that stop it from running. Would fix this myself but indentation and stuff are different right now.

**Review by William-Unosquare — APPROVED** [review]

**Review by BrownMark — APPROVED** [review]

**Pull request merged**